src/components/templates/page-template.js

```diff
@@ -1,23 +1,26 @@
 import React from 'react';
 import parse5 from 'parse5';
+import memoizee from 'memoizee';
 import PropTypes from 'prop-types';
-import faviconData from '../../../favicon-data.json';
 
-const toElementDefinition = ({ tagName, attrs }) => {
-  const props = attrs.reduce((accu, { name, value }) => {
-    accu[name] = value;
-    return accu;
-  }, {});
+const parseElementDefinitions = memoizee(html => {
+  const headElem = parse5.parse(html).childNodes[0].childNodes[0];
+  return headElem.childNodes.filter(({ nodeName }) => !nodeName.startsWith('#')).map(({ tagName, attrs }) => {
+    const attributes = attrs.reduce((accu, { name, value }) => {
+      const camelCasedName = name.replace(/-[a-z0-9]/g, c => c.toUpperCase()).replace(/-/g, '');
```

Comment:
Small thing: this would maybe be nice to have in the `string-utils`, if we ever need it again we'll probably forget we already have it here for extraction.

Reply:
Something like `convertSnakeCaseToCamelCase`? 🤔

Reply:
Good idea, moved!